grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -360,8 +361,21 @@ class StockMovementController {
                 throw new IllegalArgumentException("File must be in CSV format")
             }
 
+            String[] columnKeys = [
```

Comment:
This should be a const defined somewhere (either in consts, or StockMovement) and used in the exportCsv counterpart too

Reply:
sure I can add it, I was just following what was done here: https://github.com/openboxes/openboxes/blob/33b93dcf20c46c073d13b6335350400b7778e4c8/grails-app/services/org/pih/warehouse/order/OrderService.groovy#L896-L913

As for exportCsv, it is defined as a Map like this:
https://github.com/openboxes/openboxes/blob/33b93dcf20c46c073d13b6335350400b7778e4c8/grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy#L2757-L2769

so I'll have to think through it, if there is a reasonable way to make it reusable for both places. Would you say I should provide a change to `exportCsv` at this point as well?

Reply:
I've done it for both export/import, let me know what you think